duckdns/README.md

```diff
@@ -101,14 +100,6 @@ The number of seconds to wait before updating DuckDNS subdomains and renewing Le
 - At time of writing, Duck DNS' own IPv6 autodetection
   [does not actually work][duckdns-faq], but you can use the URL option
   for `ipv6` to get around this, read on.
-- IPv6 IP address detection service URLs are likely to not work for the time
```

Comment:
@pvizeli because this block of text was removed, the "read on" from the previous bullet should be removed too.

Reply:
@pvizeli Also, I fail to see why this block was removed entirely. IPv6 is still not available in the container, so the IPv6 services don't actually work. If you want to remove the file URL stuff, fine, but the rest of the docs that were removed are still valid as far as I can tell.